plugins/rollbar-backend/README.md

```diff
@@ -8,7 +8,6 @@ The following values are read from the configuration file.
 
 ```yaml
 rollbar:
-  organization: organization-name
```

Comment:
i wonder if it's confusing to have a pair of frontend+backend plugins, where their installation instructions (and required configs) are spread over the two readmes. I mean it kinda is what it is, just thinking out loud

Reply:
But maybe keep this line and add a comment `# used by rollbar frontend plugin`? Or is it expected that the slug variant is more common and it'd be unnecessary to mention here

Reply:
At some point I'd like to loop back and deprecate the backend in favour of using the proxy. However, there still are some open questions in general about plugin pairs. For instance, how do they share code.